docs/content/en/docs/yaml-crd-ref/app.md

```diff
@@ -128,12 +128,12 @@ metadata:
   name: podtato-head
   namespace: podtato-kubectl
 spec:
-  version: "1.3"
+  version: "latest"
   workloads:
   - name: podtato-head-left-arm
-    version: 0.1.0
+    version: "myvers12.5"
```

Comment:
maybe let's make this one my_vers-12.5 so that the one below can stay 1.2.3 so we have all possibilities 🚀